src/block/Transparent.php

```diff
@@ -23,9 +23,15 @@
 
 namespace pocketmine\block;
 
+use pocketmine\block\utils\SupportType;
+
 class Transparent extends Block{
 
 	public function isTransparent() : bool{
 		return true;
 	}
+
+	public function getSupportType(int $facing) : SupportType{
+		return SupportType::NONE();
```

Comment:
This is incorrect. Many transparent blocks are still full cubes.

Reply:
I agree on this point, however, some blocks are not real cube and do not require special support that ``NONE``.  
Instead of defining the supports of the transparent blocks one by one in ``NONE``, it is better to do the case by case of transparent blocks not having a null support ?

Reply:
No, I don't agree. For `Flowablea` this would make sense, but not for `Transparent`.